### PR TITLE
updating support policy

### PR DESCRIPTION
**Pull request by gaktive**
Per @troytop's feedback in https://github.com/SUSE/cap-release-notes/issues/81

### CODE REVIEW

**Review by btat — APPROVED** [review]
LGTM

File: adoc/support-statement.adoc [hunk 1]
```diff
@@ -39,24 +39,24 @@ https://www.suse.com/support/?id=SUSE_Cloud_Application_Platform.
...
 
-For contracted customers and partners, {product} is delivered with L3 support
-for all packages, except for the following:
+=== Platform Support
+{product} is fully supported on {eksa}, {azure} {aksa} and Google GKE. Each release is tested by {product} QA on these platforms.
 
-* Technology Previews
-* Sound, graphics, fonts and artwork
-* Packages that require an additional customer contract
+{product} is fully supported on {caaspa}, wherever it happens to be installed. If {caaspa} is supported on a particular CSP, the customer can get support for {product} in that context. (Note: This has not been tested yet but it is in theory a supportable scenario.)
```
Comment (troytop) [inline adoc/support-statement.adoc]: Omit the parenthetical comment.

File: adoc/support-statement.adoc [hunk 2]
```diff
@@ -39,24 +39,24 @@ https://www.suse.com/support/?id=SUSE_Cloud_Application_Platform.
 
 The following definitions apply:
 
-L1:: Problem determination, which means technical support designed to provide
-compatibility information, usage support, ongoing maintenance, information
-gathering and basic troubleshooting using available documentation.
+=== Version Support
+Technical Support and Troubleshooting (L1 - L2): Current and previous major versions (N-1). For example: {suse} will provide technical support and troubleshooting for versions 1.0, 1.1, 1.2, 1.3 (and all 2.x point releases) until the release of 3.0.
```
Comment (troytop) [inline adoc/support-statement.adoc]: "N-1" should probably be "_n_ -1"

File: adoc/support-statement.adoc [hunk 3]
```diff
@@ -39,24 +39,24 @@ https://www.suse.com/support/?id=SUSE_Cloud_Application_Platform.
...
 
-L2:: Problem isolation, which means technical support designed to analyze
-data, reproduce customer problems, isolate problem area and provide a
-resolution for problems not resolved by Level 1 or prepare for Level 3.
+Patches and updates (L3): On the latest or last minor release of each major release. For example: {suse} will provide patches and updates for 1.3 (and 2.latest) until the release of 3.0.
```
Comment (troytop) [inline adoc/support-statement.adoc]: maybe "2._latest_" (italic)

**Review by troytop — CHANGES_REQUESTED** [review]
A few changes required.

File: adoc/support-statement.adoc (hunk 1 above)
Comment (gaktive) [reply adoc/support-statement.adoc]: Gone.

File: adoc/support-statement.adoc (hunk 2 above)
Comment (gaktive) [reply adoc/support-statement.adoc]: Italicized & lowercased.

File: adoc/support-statement.adoc (hunk 3 above)
Comment (gaktive) [reply adoc/support-statement.adoc]: Italicized.

**Review by gaktive — COMMENTED** [review]
Updated at @troytop's request.

**Pull request merged**